### DISCUSSION TITLE
We have some conflicting ideas... let's standardize!

### DISCUSSION THREAD

**Discussion by christina-de-martinez** [christina-de-martinez/babel-plugin-glowup-vibes#37] [Announcements]
There are tons of bussin ideas and PRs, you guys are 🐐 But a lot of the ideas conflict with each other. I want to open a discussion so people can all have input on which ones should be merged. 

It's possible for two different keywords to be transpiled to the same JavaScript. For example, both `onGod` and `noCap` can become `true`. But it's not possible for the same keyword to be transpiled to two different things. So, `onGod` couldn't mean `true` and also `let`. 

### Async: [poll here](https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/discussions/38)
finna: "async" (finna was also suggested to be `expect`)
lit: "async", (lit was also suggested to be `true`)
chill: "async", (chill was also suggested to be `setTimeout` and `await`)

### Await: [poll here](https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/discussions/39)
letItCook: "await" (letHimCook was also suggested to be `continue`)
brb: "await",
holdUp: "await"

### Try/Catch/Finally
fuck around: "try",
find out: "catch",
either way: "finally"

### If/Else: [poll here](https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/discussions/40)
fr: "if", (fr is `assert` so we would need to change this to `frfr` or something?)
orfr: "else if",
nah: "else"
iykyk: "if"
jkUnless: "else if"

### Promises
gucci: 'Promise.resolve'
busted: 'Promise.reject'
bail:  'Promise.resolve'
jail: 'Promise.reject'
bet: "Promise"
vibeCheck: "Promise"
yass: "resolve" 
nah: "reject"
totes: "all"

### Other conflicts
fam: 'class' (fam was also suggested to be `Array`)

## Undisputed ideas stuck in limbo
The following ideas seem to be undisputed, but theyre stuck in limbo while we decide since the PRs include some disputed ones.

oof: 'console.error'
flex: 'console.assert'
spit: "return"
spill: 'return'
period: "break"
PERIODTTT: "break" 
finesse: 'filter'
clout: 'Math.max'
bop: 'Math.min'
fam: 'Array'
squad: 'Array'
slide: 'Array.push'
bounce: 'Array.pop'
clique: 'Object'
hundoP: '100'
sayLess: "JSON.stringify"
swag: 'JSON.stringify'
unswag: 'JSON.parse'
cash: 'parseInt'
float: 'parseFloat'
hype: 'addEventListener'
killHype: 'removeEventListener'
snack: 'fetch'
typah: "typeof"
crowed: "debug"

## Credit
Credit for these ideas goes to @LuciooF, @fastfists, @J-Palomino, @Markshall, @samueltuyizere, @stifskere, @mfairch, @henrikvtcodes, @Republique, @ismarjew, @Judge-Paul, and @aadillpickle in the following PRs and issues: 
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/pull/10/files
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/pull/6
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/pull/11
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/pull/12
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/pull/15/files
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/pull/18
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/pull/25/files
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/pull/26/files
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/issues/9
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/issues/13
https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/issues/30

As well as stickersss and prohet_isaiah on Instagram.

### Please vote and/or discuss which ones are best for each category
If there is a dedicated poll, please vote in the poll, or your vote may not be counted.

When it seems like there's some consensus, the original PR author can make the edits, or we can make new PRs with `Co-Authored-By`.

**Comment by stifskere** [comment]
I would suggest separating the object in a JSON file, so it's easier to track changes between the actual code and changes between the slang addition/changing.

**Reply by christina-de-martinez** [reply]
Good idea, I made an issue: https://github.com/christina-de-martinez/babel-plugin-glowup-vibes/issues/41

**Reply by cdxker** [reply]
#8 should solve this issue, if it helps we can remove the extra mappings we put in to be merged in later. Tests + JSON file seperation seems to be more important.

**Comment by DVG** [comment]
I kind of like `goOff` and `bet` for try/catch

```
goOff {
  yeet new L()
} bet (l){
  lowkey.sus("💀: ", l)
}
```

**Reply by henrikvtcodes** [reply]
this whole thing is a meme anyway, so por que no los dos?

**Reply by stifskere** [reply]
> this whole thing is a meme anyway, so por que no los dos?

Estoy de acuerdo.

**Comment by jackstenglein** [comment]
How about `ggGoNext` for `continue`
